### PR TITLE
skip test if the only change is markdown files

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: YangKeao <yangkeao@chunibyo.icu>

### What problem does this PR solve?

Related with https://github.com/chaos-mesh/chaos-mesh/issues/2233

### CODE REVIEW

**Review by fewdan — APPROVED** [review]
LGTM

File: ci/pingcap_chaos_mesh_build_kind.groovy [hunk 1]
```diff
@@ -220,6 +222,20 @@ def call(BUILD_BRANCH, CREDENTIALS_ID) {
...
+						it.endsWith('.md') || 
+							(!it.contains('.') && it != 'Makefile') || 
+							it.startsWith('ui') || 
+							it.startsWith('docs') || 
+							it.startsWith('static')
+					}
+					echo 'Modified Files: ' + modifiedFiles.join(',')
+					echo 'Modified Ignored Files: ' + ignoredModifications.join(',')
+					SKIP_TEST = modifiedFiles.size() == ignoredModifications.size()
+					echo String.valueOf(SKIP_TEST)
```
Comment (g1eny0ung) [inline ci/pingcap_chaos_mesh_build_kind.groovy:237]: Show more info on this echo?

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Review by g1eny0ung — APPROVED** [review]

**Pull request merged**